tfx/examples/mnist/mnist_utils_native_keras_lite.py

```diff
@@ -77,11 +77,16 @@ def run_fn(fn_args: TrainerFnArgs):
   with mirrored_strategy.scope():
     model = base.build_keras_model()
 
+  # Write logs to path specified by fn_args
+  tensorboard_callback = tf.keras.callbacks.TensorBoard(
+    log_dir=fn_args.log_dir, update_freq='batch')
+
   model.fit(
       train_dataset,
       steps_per_epoch=fn_args.train_steps,
       validation_data=eval_dataset,
-      validation_steps=fn_args.eval_steps)
+      validation_steps=fn_args.eval_steps,
+      callbacks=[tensorboard_callback])
```

Comment:
could you try run the e2e test next to those examples to make sure everything works correctly?

Reply:
e2e tests pass

Reply:
could you also run a e2e example, and check the ~/tfx outputs and see if tensorboard works correctly